.github/workflows/package-event-processing.yaml

```diff
@@ -0,0 +1,53 @@
+name: Package event processing infrastructure
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - infrastructure/event-processing/**
+  workflow_dispatch: {}
+
+env:
+  AWS_REGION: eu-west-2
+
+jobs:
+  package:
+    strategy:
+      matrix:
+        include:
+          - target: EVENT_PROCESSING_DEV
+            ARTIFACT_BUCKET_NAME_SECRET: ARTIFACT_BUCKET_NAME_CORE_EVENT_PROCESSING_DEV
```

Comment:
this bucket `ARTIFACT_BUCKET_NAME_CORE_EVENT_PROCESSING_DEV` is used to deploy the core infra `infrastructure/core/template.yaml`. 

we need to create 3 new secrets and deploy the terraform infrastructure that creates the dev platform deployment pipelines https://github.com/govuk-one-login/txma-pipelines

if we reuse the bucket, we will be overwritting the core infra